Objects/structseq.c

```diff
@@ -580,7 +580,8 @@ _PyStructSequence_FiniType(PyTypeObject *type)
     assert(type->tp_base == &PyTuple_Type);
 
     // Cannot delete a type if it still has subclasses
-    if (type->tp_subclasses != NULL) {
+    if (_PyType_HasSubclasses(type)) {
+        PyErr_Clear();
```

Comment:
Where this exception is raised?

Reply:
Good catch.  `lookup_subclasses()` could fail before.  Now it can't.  I forgot to update all the call sites.  Now I have.